packages/electrode-react-webapp/lib/hapi/index.js

```diff
@@ -71,4 +71,9 @@ registerRoutesPlugin.attributes = {
   }
 };
 
-module.exports = registerRoutesPlugin;
+// registerRoutesPlugin.registerRoutes = registerRoutes;
+
+module.exports = {
```

Comment:
this definitely going to break peoples that use hapi directly.

Reply:
It was always broken.